### ISSUE TITLE
fix accessibility issues in the giveth stack

### ISSUE THREAD

**Issue by alibama** [Giveth/giveth-next#365]
this is a small request that is also meant to help me get a better sense of how giveth works by contributing some little fixes to the code.  to begin with i want to try to get a baseline of 508 wcag compliance - it's a basic standard and aside from making the site more usable it should help with search engine ranking as well.... not that it's a major concern, however it doesn't hurt :)

https://github.com/alibama/giveth-next is the fork i'm testing against

## 📋 Anything to add?
the site is not in great shape for blind and low-vision users.  most of the changes are simple - adding alt tags for images and making sure the buttons on the site are properly labeled with aria tags, for instance.  i work with the national federation of the blind and would like to be able to show this kind of site to them when it's ready

## ⏰ Urgency/Deadline/Blockers?
not urgent - most changes should have no impact on sited users, and there are no blockers or deadlines to my knowledge

## ❓ Why is it important?
addressing accessibility early in the development process makes it a lot easier to maintain in the long run and opens up opportunities.  multi-language sites should be next on the list.  also if you ever want to work with state, local, or federal governmental agencies it is a requirement.  https://www.section508.gov/sell/vpat/ is the general template for VPAT's that are required for softwares used in the public space

## 🎉 Subtasks
make revisions manually as needed
test axe with cypress and axe https://www.npmjs.com/package/cypress-axe for standardizing approach?

## 🤼‍ Reviewer

@

**Comment by mateodaza** [comment]
This is excellent feedback!! we'll have this in our bucket for later. Thanks
